docs/source/intro/get_started.rst

```diff
@@ -4,11 +4,11 @@
 Getting Started
 ****************
 
-A working AiiDA installation consists of three core components, plus any external codes you wish to run:
+An AiiDA installation consists of three core components (plus any external codes you wish to run):
 
-* aiida-core: The main Python package and the associated command line tool: ``verdi``.
-* |PostgreSQL|: A service which manages the database that AiiDA uses to store generated data.
-* |RabbitMQ|: A service which manages communication with the processes that AiiDA spawns.
+* ``aiida-core``: The main Python package and the associated ``verdi`` command line interface
+* |PostgreSQL|: The service that manages the database that AiiDA uses to store data.
+* |RabbitMQ|: The message broker used by the AiiDA daemon to manage your simulations.
```

Comment:
We don't just manage "simulations".

Reply:
It is also not correct. I get that we don't want to make it too technical, but we also don't want to give the wrong impression. How about:
> The message broker that enables communication with and between AiiDA's process runners
Concise, correct and understandable, no? Even if "process runner" may not be fully understood, I think the majority of people will also not understand "message broker" but that is not important here I would say

Reply:
> We don't just manage "simulations".

Happy to improve - what would you suggest instead? Workflows?
Please note that "simulations" are actually mentioned twice in the abstract https://arxiv.org/abs/2003.12476

> Even if "process runner" may not be fully understood,

I agree that "process runner" will not be understood, and I therefore don't think we should use the term here, which is the entry point for every new user installing AiiDA.
In my opinion, a new user should not even have to care about the fact that there are different runners, until the point where they need to do performance tuning.
All they need to know here is that rabbitmq is used by the daemon to keep track of what it does.

>  I think the majority of people will also not understand "message broker" but that is not important here I would say

I accepted your suggestion to replace "service" by "message broker" because I felt it was a more descriptive term, and one that is used outside AiiDA as well.
But if we think people will not understand what it means, we should go back to "service".

Reply:
"The message broker used for communication within AiiDA"?

Reply:
> In my opinion, a new user should not even have to care about the fact that there are different runners, until the point where they need to do performance tuning.

I agree that they don't need to understand exactly what it does at this point, so a strongly reduced description is fine. Changing "service" with "message broker" was also not the real point, just that the rest of the statement was incorrect...

> All they need to know here is that rabbitmq is used by the daemon to keep track of what it does.

... as is this. It is not at all used "by the daemon to keep track of what it does". Brevity is fine, but not at the cost of correctness.

But I am fine with the suggestion of @greschd